src/components/reusable/tabs/tabs.ts

```diff
@@ -124,13 +129,14 @@ export class Tabs extends LitElement {
    * @param {string} selectedTabId - The selectedTabId parameter is a string that represents the ID of
    * the tab that is currently selected.
    */
-  private _updateChildrenSelection(selectedTabId: string) {
+  private _updateChildrenSelection(selectedTabId: string, updatePanel = true) {
     // update tabs selected prop
     this._tabs.forEach((tab: any) => {
       tab.selected = tab.id === selectedTabId;
     });
 
     // update tab-panels visible prop
+    if (!updatePanel) return;
```

Comment:
You could just reference the new prop directly here rather than abstracting it down as a parameter of the update children function.

Reply:
@brian-patrick-3 , but _updateChildrenSelection should be able to differentiate if it is called on Enter/Space key  Vs left/right arrows on both cases of disableAutoFocusUpdate

Reply:
Gotcha, ok. Sorry hard to tell on a phone!